server/src/metadata.js

```diff
@@ -228,10 +226,17 @@ class Metadata {
                      res('new_val'))).run(this._conn)
       ]);
     }).then(() => {
+      logger.debug('redirecting users table');
       // Redirect the 'users' table to the one in the internal db
       this._collections.set('users', new Collection({ id: 'users', table: 'users' },
                                                     this._internal_db, this._conn));
-    }).catch((e) => {
+    }).then(() => {
+      logger.debug('metadata sync complete');
+      this._ready = true;
+      return this;
+    })
```

Comment:
Yes, the semicolons are optional - we actually have a bunch of eslint style complains after last night's adventures that I'll get to soon, but they're not too important.